survey-sdk/src/main/java/co/nimblehq/survey/sdk/network/NetworkBuilder.kt

```diff
@@ -0,0 +1,68 @@
+package co.nimblehq.survey.sdk.network
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+
+class NetworkBuilder {
+    private var debugMode = false
+    private var baseUrl = ""
+    private var connectionTimeoutInSecond = 30L
+    private var readTimeoutInSecond = 30L
+    private var version = "v1"
+
+    fun setDebugMode(debugMode: Boolean): NetworkBuilder {
+        this.debugMode = debugMode
+        return this
+    }
+
+    fun setBaseUrl(url: String): NetworkBuilder {
+        this.baseUrl = url
+        return this
+    }
+
+    fun setConnectionTimeoutInSecond(timeout: Long): NetworkBuilder {
+        this.connectionTimeoutInSecond = timeout
+        return this
+    }
+
+    fun setReadTimeoutInSecond(timeout: Long): NetworkBuilder {
+        this.readTimeoutInSecond = timeout
+        return this
+    }
+
+    fun setVersion(version: String): NetworkBuilder {
+        this.version = version
+        return this
+    }
+
+    fun provideRetrofit(
+    ): Retrofit {
```

Comment:
broken formatting, it could be in 1 line

Reply:
@luongvo Updated